python/taichi/core/util.py

```diff
@@ -251,17 +251,19 @@ def check_exists(src):
         )
 
 
-def prepare_sandbox(src):
+def prepare_sandbox(src=None):
     global g_tmp_dir
-    check_exists(src)
+    if src is not None:
+        check_exists(src)
```

Comment:
We always need a sandbox for CC to store temp files, even in release mode.

Reply:
OK, that's totally fine. Could you add this as a comment to the code?